plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/ToServlet25SourceQuickFixTest.java

```diff
@@ -68,12 +58,10 @@ public void testConvertServlet() throws CoreException {
     IDocument document = viewer.getDocument();
     String contents = document.get();
     assertFalse(contents.contains("version='3.1'"));
+    assertTrue(contents.contains("version=\"2.5\""));
 
     // https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1527
     editorPart.doSave(new NullProgressMonitor());
-
-    ProjectUtils.waitForProjects(project);
-    assertEquals(0, file.findMarkers(SERVLET_MARKER, true, IResource.DEPTH_ZERO).length);
```

Comment:
The justification of removing this code being that it's an end-to-end test, and this outcome is already tested elsewhere.  Right?  Seems reasonable, just want to verify.

Reply:
This is not tested elsewhere actually, but I did want to downgrade this end-to-end test to a smaller unit test. I think it's a reasonable change considering the new assertion.